velox/common/encode/Base64.h

```diff
@@ -32,125 +29,100 @@ class Base64 {
   static const size_t kCharsetSize = 64;
   static const size_t kReverseIndexSize = 256;
 
-  /// Character set used for encoding purposes.
-  /// Contains specific characters that form the encoding scheme.
+  /// Character set used for Base64 encoding.
```

Comment:
@Joe-Abraham why remove `/// Contains specific characters that form the encoding scheme.`?